src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figureSupplier.getRandomFigure().printInformation();
+                continue;
+            }
+            figureSupplier.getDefaultFigure().printInformation();
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. However, you are not creating a new FigureSupplier, but you are not storing the created figures in the array. You should assign the created figures to the `figures` array elements instead of just printing them out.